.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: swift
+osx_image: xcode9.3
+xcode_project: Static.xcodeproj
+
+script: xcodebuild -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK" -destination "$DESTINATION" test
+
+matrix:
+  include:
+    - xcode_scheme: Static-iOS
+      xcode_sdk: iphonesimulator
+      env:
+        - DESTINATION="OS=10.1,name=iPhone 7 Plus"
```

Comment:
It might be good to test multiple sims, but this is a fine start.

Reply:
@eliperkins Long time, Thanks for the feedback didn't realize this was missing CI for so long. 

What issues have you run into while testing which require multiple sims? Size vs OS version?

Reply:
I think mxcl/PromiseKit has an extremely thorough `.travis.yml` that covers most bases: https://github.com/mxcl/PromiseKit/blob/98eaf180ed8ff5246f36897b9c088d7a8fecaf58/.travis.yml

I don't think Static needs all these things, but testing against supported Swift versions, as well as against iOS versions is a good idea, especially for a UI-based project like this.

Reply:
I don't think different sim devices is a big deal right now, since we don't do snapshot testing or anything that's concerned with the size of the device really.

Reply:
Thanks!